app/views/users/show.html.erb

```diff
@@ -1 +1,32 @@
-<h1>Showing a specific User</h1>
\ No newline at end of file
+<%= stylesheet_link_tag 'users_index' %>
+<%= stylesheet_link_tag 'users_show' %>
+<section class='user-section'>
+    <div class='user-container'>
+        <%= image_tag(@user.photo) %>
+        <div class='user-info'>
+            <h2 class='username'><%= @user.name %></h2>
+            <p class='number-of-posts'>Number of posts: <%= @user.posts_counter %></p>
+        </div>
+    </div>
+        
+    <div class='user-bio'>
+        <h4>Bio</h4>
+        <p><%= @user.bio %></p>
+    </div>
+
+    <div class='user-posts'>
+        <h2>Posts</h2>
+        <% @user.recent_posts.each_with_index do |post, index| %>
+            <div class='post-container'>
+                <h3><%= post.title %></h3>
+                <p><%= post.text %></p>
+                <div class='post-interactions'>
+                    <p>Comments: <%= post.comments_counter %></p>
+                    |
+                    <p>Likes: <%= post.likes_counter %></p>
+                </div>
+            </div>
+        <% end %>
+    </div>
+</section>
+
```

Comment:
- [ ] As required for the project and  to match the given wireframe, kindly ensure that your `app/views/users/show.html.erb` page has a **See all Post** link/button and when cliking on it it opens `user's post index` as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png)